lua/nvim-lsp-setup/init.lua

```diff
@@ -57,7 +57,12 @@ function M.setup(opts)
     require('mason-lspconfig').setup_handlers({
         function(server_name)
             local config = servers[server_name] or {}
-            require('lspconfig')[server_name].setup(config)
+            local ok, coq = pcall(require, 'coq')
+            if ok then
+                require('lspconfig')[server_name].setup(coq.lsp_ensure_capabilities(config))
+            else
+                require('lspconfig')[server_name].setup(config)
+            end
```

Comment:
```suggestion
            if ok then
                config = coq.lsp_ensure_capabilities(config)
            end
```

Reply:
Thanks, I gave a little suggestion : )

Reply:
Yes, good suggestion. Thank you :smiley: